ood-portal-generator/bin/generate

```diff
@@ -5,6 +5,8 @@ SCL_SOURCE=$(command -v scl_source)
 
 ROOT_DIR="$(dirname "$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")")"
 
+[ -f /etc/os-release ] && source /etc/os-release
```

Comment:
This isn't making it into the environment for some reason. The source for SCL is in ENV when I dump it but not the contents of /etc/os-release.

Reply:
I think because the environment variables in `/etc/os-release` don't use `export`, they are not accessible except in script that sourced them